code/modules/projectiles/updated_projectiles/guns/pistols.dm

```diff
@@ -91,8 +91,8 @@
 //M4A3 45 //Inspired by the 1911
 
 /obj/item/weapon/gun/pistol/m1911
-	name = "\improper M4A3 service pistol (.45)"
-	desc = "A standard M4A3 chambered in .45. Has a smaller magazine capacity, but packs a better punch."
+	name = "\improper Concord FourFive service pistol"
```

Comment:
Prefer numerical symbols over their literary equivalent when naming things in most cases, for both length (and caliber, in these cases).